src/python_testing/matter_testing_support.py

```diff
@@ -417,8 +417,23 @@ class CustomCommissioningParameters:
 
 
 @dataclass
-class AttributePathLocation:
+class ProblemLocation:
```

Comment:
A dataclass without members feels odd?

How about defining a type instead of 

ProblemLocation = typing.Union[ClusterPathLocation, DeviceTypeLocation]

since the only use of this seems to be typings.

Reply:
yeah, I think I originally had the endpoint in there and then changed my mind. union is fine too - changed.